Packs/CrowdStrikeMalquery/Playbooks/CrowdStrikeMalquery_-_GenericPolling_-_Search.yml

```diff
@@ -0,0 +1,422 @@
+      name: GenericPolling
+      description: |-
+        Use this playbook as a sub-playbook to block execution of the master playbook until a remote action is complete.
+        This playbook implements polling by continuously running the command in Step \#2 until the operation completes.
+        The remote action should have the following structure:
+
+        1. Initiate the operation.
+        2. Poll to check if the operation completed.
+        3. (optional) Get the results of the operation.
+      playbookName: GenericPolling
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      AdditionalPollingCommandArgNames: {}
+      AdditionalPollingCommandArgValues: {}
+      Ids:
+        simple: ${Malquery.Request_ID}
+      Interval:
+        simple: "1"
+      PollingCommandArgName:
+        simple: request_id
+      PollingCommandName:
+        simple: cs-malquery-get-request
+      Timeout:
+        simple: "10"
+      dt:
+        simple: Malquery(val.Status == 'inprogress').Request_ID
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
+    id: "6"
+    taskid: 0a845a62-4e09-4232-8da8-c961e915e7f4
+    type: title
+    task:
+      id: 0a845a62-4e09-4232-8da8-c961e915e7f4
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 895
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "7":
+    id: "7"
+    taskid: 05ff1310-f8bd-420f-8d57-73a227b9dea2
+    type: regular
+    task:
+      id: 05ff1310-f8bd-420f-8d57-73a227b9dea2
+      version: -1
+      name: Get request data
+      description: Check the status and results of an asynchronous request, such as
+        hunt or exact-search.  Supports a single request id at this time.
+      script: '|||cs-malquery-get-request'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      request_id:
+        simple: ${Malquery.Request_ID}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 910,
+        "width": 810,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: search command
```

Comment:
@yaron-libman good catch, thanks.
it will work but better to keep up with the standard.

@Bargenish please rename it (use `snake_case`)

Reply:
Done.
@yaron-libman thanks!